py-polars/tests/unit/operations/namespaces/test_plot.py

```diff
@@ -1,40 +1,34 @@
-from datetime import date
-
-import pytest
-
 import polars as pl
 
-# Calling `plot` the first time is slow
-# https://github.com/pola-rs/polars/issues/13500
-pytestmark = pytest.mark.slow
```

Comment:
importing Altair is about 70 times faster* than importing hvplot, so I think we can remove this slow marker

*timed by performing `time python -c 'import altair'` and `time python -c 'import hvplot'` 7 times each, and finding the ratio of the smallest "real time" results for both